lib/kissable/ab.rb

```diff
@@ -19,8 +19,10 @@ def initialize(test_name, groups=nil, ratios=nil)
       validate_ratios
     end
 
-    def group(cookies)
-      @cookies = cookies
+    # Assigns a test group based on the 'abid' cookie or login (email).
+    # Takes either a cookies object (Hash-like) or login (String).
+    def group(object)
+      (object.is_a? String) ? (@login = object) : (@cookies = object)
```

Comment:
@mattysads Can you verify this question: Will we ever have the case where we want the cookie to override the login we pass in?